src/lib/tracking.ts

```diff
@@ -37,8 +38,21 @@ export const addTrackingParams = (
     };
 
     const queryString = Object.entries(trackingLinkParams).map(([key, value]) => `${key}=${value}`);
+    const alreadyHasQueryString = baseUrl.includes('?');
 
-    return `${baseUrl}?${queryString.join('&')}`;
+    return `${baseUrl}${alreadyHasQueryString ? '&' : '?'}${queryString.join('&')}`;
+};
+
+export const addRegionIdAndTrackingParamsToSupportUrl = (
```

Comment:
it would be good to add a couple of tests to tracking.test.ts for this function
e.g. where
- baseUrl has a querystring
- baseUrl isn't support.theguardian.com
happy to pair on this

Reply:
yeah for sure! I've added tests to
- check `addTrackingParams` handles existing query strings
- check `addRegionIdAndTrackingParamsToSupportUrl` doesn't do anything for a non-support url
- check `addRegionIdAndTrackingParamsToSupportUrl` works correctly for a suppor url (adds region ID + tracking params)